runtime/foucoco/src/lib.rs

```diff
@@ -1293,18 +1293,30 @@ where
 				let mut env = env.buf_in_buf_out();
 				let price_feed_request: ([u8; 32], [u8; 32]) = env.read_as()?;
```

Comment:
Also here, let's maybe introduce types for the byte arrays to improve readability

Reply:
done 👍